tests/conftest.py

```diff
@@ -43,7 +43,7 @@ def temp_path():
 
 @pytest.fixture
 def test_item(tgtg_item: dict):
-    return Item(tgtg_item)
+    return Item(tgtg_item, Config())
```

Comment:
Sorry, I had to update the tests due to a bug that modified the `DEFAULT_CONFIG` between tests.

If you could avoid passing the config to the Item object, this change is obsolete of course.

```suggestion
    reload(models.config)
    return Item(tgtg_item, models.config.Config())
```

Reply:
No need, indeed